### PR TITLE
Write history to s3

### PR DESCRIPTION
**Pull request by graydon**

### CODE REVIEW

**Comment by matschaffer** [comment]
If we merge this as-is we may want to file an issue to clean this up for later since this will effectively prohibit doing scc docker runs without S3 (fine for us but maybe not everyone who'll want to run this stuff).

File: lib/stellar_core_commander/docker_process.rb [hunk 1]
```diff
@@ -216,14 +226,12 @@ def config
...
-        PREFERRED_PEERS=#{peers}
+        PREFERRED_PEERS=#{peer_connections}
         QUORUM_SET=#{quorum}
 
-        HISTORY_PEERS=["main"]
-
-        HISTORY_GET=cp history/%s/{0} {1}
-        HISTORY_PUT=cp {0} history/%s/{1}
-        HISTORY_MKDIR=mkdir -p history/%s/{0}
+        HISTORY_PEERS=#{peer_names}
```
Comment (nullstyle) [inline lib/stellar_core_commander/docker_process.rb]: Should be pretty easy to make this conditional, where we fallback to the local history if the s3 cli arguments are blank, right?

**Comment by nullstyle** [comment]
:+1:

File: lib/stellar_core_commander/docker_process.rb [hunk 2]
```diff
@@ -60,6 +60,7 @@ def shutdown_heka_container
     Contract None => Any
     def write_config
       IO.write("#{working_dir}/stellar-core.env", config)
+      IO.write("/tmp/stellar-core.env", config)
```
Comment (matschaffer) [inline lib/stellar_core_commander/docker_process.rb]: Should this still be here?

Comment (graydon) [inline lib/stellar_core_commander/docker_process.rb]: oh oops! yes that was a debug aid, sorry. removing.

**Pull request merged**